### PR TITLE
product_description_generator notebook

### PR DESCRIPTION
**Pull request by lavinigam-gcp**
product_description_generator notebook 
- cleared output
- ran through code quality checked
- fixed links and added markdowns
- added new examples 
- removed hardcoded endpoint and project_ids

### CODE REVIEW

File: language/examples/document-qa/question_answering_large_documents.ipynb [hunk 1]
```diff
@@ -174,11 +174,11 @@
    },
```
Comment (erwinh85) [inline language/examples/document-qa/question_answering_large_documents.ipynb:174]: <p>Can you add for colab: 'or use the button to restart'</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/47/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Comment (lavinigam-gcp) [reply language/examples/document-qa/question_answering_large_documents.ipynb:174]: added.

**Comment by lavinigam-gcp** [comment]
<p>removed. </p>

---

 <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/47/discussion/#comment-1584040735'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by lavinigam-gcp** [comment]
<p>Agreed. Fixed. </p>

---

 <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/47/discussion/#comment-1584040743'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Review by erwinh85 — APPROVED** [review]
LGTM

**Pull request closed without merging**